### DISCUSSION TITLE
European general requirements for future IWXXM schema

### DISCUSSION THREAD

**Discussion by jnuottokari** [team-sx/iwxxm-sx#5] [General]
The European MET Service Providers have agreed on the following for the general elements of the MET SWIM services: 

Aerodrome Meteorological Observation Information Service containing the information currently included in the METAR, SPECI, local routine report and local special report as a minimum requirement but allowing for more temporal resolution and mutliple data points along runways

Aerodrome Meteorological Forecast Information Service, containing the information currently included in the TAF and TREND as a minimum requirement but allowing for more temporal resolution and additional parameters, such as LVP%, CB% and runways-specific wind forecasts. 

WAFC Information Services for GRIB and SIGWx 

Enroute Significant Weather Information Service, implemented by all MWOs, containing the information currently included in the SIGMET as a minimum requirement, but allowing for additional information elements 

Quantitative Volcanic Ash information service

**Comment by blchoy** [comment]
Thanks Jaakko.  May I ask if the Enroute Significant Weather Information Service is expected to be superseded by HWIS?

**Comment by blchoy** [comment]
Closed because it has been dormant for quite some time.
